tests/tests.yaml

```diff
@@ -6,6 +6,6 @@ smoke_bundles:
   - bionic-queens-swift-v3
```

Comment:
The ``tests`` directory lives under the ``src`` subdirectory in reactive charms.

Reply:
moved